psamm/datasource/sbml.py

```diff
@@ -773,6 +773,13 @@ def _add_gene_list(self, parent_tag, gene_id_dict):
             gene_tag.set(_tag('id', FBC_V2), id)
             gene_tag.set(_tag('label', FBC_V2), label)
 
+    def _transform_param(self, stri):
+        if '.' in stri:
+            stri = stri.replace('.', '_')
+        if '-' in stri:
+            stri = stri.replace('-', 'neg')
+        return stri
```

Comment:
There's no point in doing `if character in stri` when you call `replace` afterwards. `replace` is safe to call even if the character is not in `stri`. This means that the method can be written simply as
``` python
def _transform_param(self, stri):
    return stri.replace('.', '_').replace('-', 'neg')
```

Also, maybe change the name to reflect that this method is specifically for flux bound parameters? or at least for numerical values... (maybe `_make_safe_numerical_id` or something like that)